xbmc/interfaces/legacy/InfoTagVideo.h

```diff
@@ -610,6 +611,61 @@ namespace XBMCAddon
 #else
       unsigned int getDuration();
 #endif
+
+#ifdef DOXYGEN_SHOULD_USE_THIS
+      ///
+      /// \ingroup python_InfoTagVideo
+      /// @brief \python_func{ getUniqueID(key) }
+      ///-----------------------------------------------------------------------
+      /// To get the uniqueID of the given key
```

Comment:
Not having context of what a "uniqueID" is, maybe give examples of what a unique ID is used for? It also isn't clear from the defaults listed below.

Reply:
I'll add a short description of what a unique ID is.